ipn/ipnlocal/local_test_helpers.go

```diff
@@ -0,0 +1,94 @@
+// Copyright (c) Tailscale Inc & AUTHORS
```

Comment:
what's with this filename? Are these only for tests? If so, it should be helpers_test.go.